src/gwt/src/org/rstudio/studio/client/RStudio.java

```diff
@@ -290,6 +291,7 @@ private void ensureStylesInjected()
       CppCompletionResources.INSTANCE.styles().ensureInjected();
       RSConnectDeploy.RESOURCES.style().ensureInjected();
       NewConnectionShinyHost.ensureStylesInjected();
+      RStudioAPI.ensureStylesInjected();
```

Comment:
As an alternative, you might just want to call `ensureInjected()` within a static block, e.g.

https://github.com/rstudio/rstudio/blob/master/src/gwt/src/org/rstudio/studio/client/projects/ui/newproject/NewDirectoryNavigationPage.java#L194-L195

I'd argue this solution is nicer (assuming it works in your use case) since the changes required can exist entirely within one file.